src/components/Preview.jsx

```diff
@@ -34,6 +34,8 @@ class Preview extends React.Component {
         nonBlockingAlertsAndPrompts: isLivePreview,
       }
     ).documentElement.outerHTML;
+
```

Comment:
Yup gc, there was no deep intent behind the fact that `generatePreview` was returning a DOM, makes more sense the way you’ve got it now.